cmd/rekor-cli/app/get.go

```diff
@@ -203,6 +203,10 @@ func compareEntryUUIDs(requestEntryUUID string, responseEntryUUID string) error
 }
 
 func parseEntry(uuid string, e models.LogEntryAnon) (interface{}, error) {
+	if viper.GetString("format") == "bundle" {
```

Comment:
Why about "proto"? "bundle" should be used for returning https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto, otherwise it's overloaded